### PR TITLE
feat(message-system): 46 update TS3 wipe warning

### PR DESCRIPTION
**Pull request by matejkriz**
Notion link: https://www.notion.so/satoshilabs/update-TS3-wipe-warning-release-date-8c4a479442144f0fb48e42e7034eb3b0?pvs=4

## Description

**This should be published right after 23.11 public release.** 
_In case the public release is made after 2023-11-15T12:00:00, this configuration would still need to be modified._

- Remove duration condition for TS3 warnings for wipe during FW upgrade 2.6.2 -> 2.6.3.

Update of https://github.com/trezor/trezor-suite/pull/9746

### CODE REVIEW

File: suite-common/message-system/config/config.v1.json [hunk 1]
```diff
@@ -336,14 +290,14 @@
             "conditions": [
                 {
                     "environment": {
-                        "desktop": "<23.7.2",
+                        "desktop": "<23.9.3",
```
Comment (komret) [inline suite-common/message-system/config/config.v1.json:293]: Why this change?

**Review by komret — APPROVED** [review]

File: suite-common/message-system/config/config.v1.json (hunk 1 above)
Comment (matejkriz) [reply suite-common/message-system/config/config.v1.json:293]: just a regular bump, see https://www.notion.so/satoshilabs/Secure-message-system-5ade512cbd38481f9d5f65f03dbfd9a8?pvs=4#1d551cfbac9d49218cee628a99426212
